go.mod

```diff
@@ -68,10 +68,10 @@ require (
 	go.etcd.io/bbolt v1.3.8
 	golang.org/x/exp v0.0.0-20231006140011-7918f672742d
 	golang.org/x/net v0.25.0
-	golang.org/x/sync v0.12.0
-	golang.org/x/sys v0.31.0
-	golang.org/x/term v0.30.0
-	golang.org/x/text v0.23.0
+	golang.org/x/sync v0.11.0
+	golang.org/x/sys v0.30.0
+	golang.org/x/term v0.29.0
+	golang.org/x/text v0.22.0
```

Comment:
I merged the first one, but just realized these are going downwards and some of them we have had other CVE fixes for.  Is there anyway to redirect to the OCP repo, but not ding these?

Reply:
No, keep in mind these are downgrades because we updated to the latest and now we have to go back to earlier versions so they don't require go 1.23.

The diff here is in a sense lying, you should compare the base from https://github.com/containers/podman/pull/25540
```
golang.org/x/net v0.18.0
golang.org/x/sync v0.6.0
golang.org/x/sys v0.20.0
golang.org/x/term v0.17.0
golang.org/x/text v0.14.0
```

All them are still upgraded several versions. So unless you go CVE reports before that we ignored this will not add any new CVE's to this branch

Reply:
Phew, thx @Luap99